bom/application/pom.xml

```diff
@@ -24,7 +24,7 @@
         <opentracing-jdbc.version>0.2.4</opentracing-jdbc.version>
         <opentracing-kafka.version>0.1.15</opentracing-kafka.version>
         <opentracing-mongo.version>0.1.5</opentracing-mongo.version>
-        <opentelemetry.version>1.16.0</opentelemetry.version>
+        <opentelemetry.version>1.17.0</opentelemetry.version>
         <opentelemetry-alpha.version>1.16.0-alpha</opentelemetry-alpha.version>
```

Comment:
We need to wait for this one to be released so that versions are consistent.

Reply:
Nice trick but it wasn't when you posted it :).

Reply:
done